DJHelper/DJHelper/EventPageViewController.swift

```diff
@@ -43,20 +62,55 @@ class EventPageViewController: UIViewController {
     }
 
     // MARK: - Private functions
-    func setupButtons() {
+    private func setupButtons() {
         detailButtonProperties.colorTheme()
         shareLinkButtonProperties.colorTheme()
         addSongButtonProperties.colorTheme()
     }
-    /*
-    // MARK: - Navigation
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    // Provides feature to send a URL via the Mail app.
+    // When testing, it only works on a device, not the simulator.
+    private func setupEmailForLink() {
+        guard MFMailComposeViewController.canSendMail(),
```

Comment:
I've never heard of this class - it's good to be exposed to new things! Did you look at UIActivityViewController? I think if we are unhappy with the MFMailComposeVC we can use the activityVC as a backup.